internal/app/frontend/frontend_service_test.go

```diff
@@ -340,6 +340,88 @@ func TestDoWatchAssignments(t *testing.T) {
 	}
 }
 
+// TestAcknowledgeBackfill - test input validation
+func TestAcknowledgeBackfillValidation(t *testing.T) {
+	cfg := viper.New()
+	tests := []struct {
+		description string
+		request     *pb.AcknowledgeBackfillRequest
+	}{
+		{
+			description: "check backfillId exists",
```

Comment:
Please, provide a more informative test case description.
Follow the pattern - "what you test, what you expect"
Example: "no backfillID, err expected"